Update sizes-hpc.md

mabicca opened the pull request:
Added note about our InfiniBand implementation which does not currently support IP over InfiniBand:

> [!NOTE]
> Currently, Infiniband does not support IP over InfiniBand (IPoIB)

azurecla commented:
Hi __@JamieDunstan__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

rmca14 commented:
@mimig1

mimig1 commented:
#sign-off @rmca14 Thanks Jamie!

mabicca committed:
Update sizes-hpc.md

Added note about our InfiniBand implementation which does not currently support IP over InfiniBand:

> [!NOTE]
> Currently, Infiniband does not support IP over InfiniBand (IPoIB)
